syntax = "proto3";

//define serial service
service Serial {
  rpc GetSerial (PlaceHolder) returns (SerialNumber) {}
}

//define map request message
message PlaceHolder {
  string place_holder = 1;
}

//response message containing the map features
message SerialNumber {
  string serial_no = 1;
}